en/docs/integrations/build-your-first-integration.md

```diff
@@ -35,7 +35,7 @@ Follow this procedure to connect to the COVID-19 API and retrieve data:
 5. Now let’s extract the total case count from the response and store it in a variable. Follow this procedure:
 
     1. Click the last **+** icon in the low-code diagram.
-    2. Under **Statements**, select **Variable** and enter details as follows:
+    2. Under **Statements**, select **Variable** and enter these details:
```

Comment:
same

Reply:
Since "enter the following details" and "enter these details" are both correct and the copy editor’s suggestion has a less number of words, I did not go ahead and change.